build-logic/src/main/kotlin/designcompose/conventions/roborazzi.gradle.kts

```diff
@@ -38,7 +38,7 @@ project.pluginManager.withPlugin("com.android.base") {
                         showExceptions = true
                         showStackTraces = true
                         showCauses = true
-                        showStandardStreams
+                        showStandardStreams = true
```

Comment:
Show stdout in tests to help with debugging.

I swear I've turned this on in five seperate branches now